module/userProjects/helper.js

```diff
@@ -1236,6 +1237,10 @@ module.exports = class UserProjectsHelper {
 
                         if( bodyData.referenceFrom ) {
                             projectCreation.data.referenceFrom = bodyData.referenceFrom;
+                            // Add link to project Details
+                            if( bodyData.referenceFrom == CONSTANTS.common.LINK && bodyData.link && bodyData.link != "" ) {
```

Comment:
@VISHNUDAS-tunerlabs  is frontend passing these values in the body please confirm and check of old APP as well

Reply:
@ankitpws If they send only we will store, But in the new flow for project link, front end has to send the link